pkg/sampling/w3ctracestate_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sampling
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseW3CTraceState(t *testing.T) {
+	type testCase struct {
+		in        string
+		rval      string
+		tval      string
+		extra     map[string]string
+		expectErr error
+	}
+	const ns = ""
+	for _, test := range []testCase{
+		// correct cases
+		{"ot=th:1", ns, "1", nil, nil},
+		{" ot=th:1 ", ns, "1", nil, nil},
+		{"ot=th:1", ns, "1", nil, nil},
```

Comment:
isn't this the same as the first case?

Reply:
Looks like a copied block of code, oops. I've removed 3 or 4 duplicates, leaving slight variations in whitespace.